packages/core-mobile/app/store/notifications/listeners/utils.ts

```diff
@@ -5,14 +5,21 @@ import { selectNotificationSubscription } from '../slice'
 
 export const isStakeCompleteNotificationDisabled = async (
   listenerApi: AppListenerEffectAPI
-) => {
+): Promise<undefined | boolean> => {
   const state = listenerApi.getState()
 
   const isInAppNotificationEnabled = selectNotificationSubscription(
     ChannelId.STAKING_COMPLETE
   )(state)
 
+  const isStakeCompleteNotificationBlocked = async (): Promise<
+    boolean | undefined
+  > => {
+    const blockedNotifications =
+      await NotificationsService.getBlockedNotifications()
+    return blockedNotifications.get(ChannelId.STAKING_COMPLETE)
```

Comment:
can we simply `return !!blockedNotifications.get(ChannelId.STAKING_COMPLETE)` to make the return type only as boolean?

Reply:
[No undefined return](https://github.com/ava-labs/core-mobile/pull/1659/commits/848ec785e8f852da740094f62635549b35a8b7fd)